### PR TITLE
Restructure Stream Data/Changefeed Navigation

### PR DESCRIPTION
**Pull request by kathancox**
Fixes [DOC-1792](https://cockroachlabs.atlassian.net/browse/DOC-1792)

Broken down the current Stream Data Out of CockroachDB Using Changefeeds page into separate pages under the new top-level Stream Data nav header.

### CODE REVIEW

**Comment by kathancox** [comment]
Hey @mikeCRL and @ianjevans, here is my first draft of the CDC docs restructure project. It would be great if you could give this an initial review!

File: v21.2/monitor-and-debug-changefeeds.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+summary: Monitor a changefeed from the DB console and use logs for debugging.
+toc: true
+---
+
+Changefeeds work as jobs in CockroachDB, which allows for [monitoring](#monitor-a-changefeed) and [debugging](#debug-a-changefeed) through the [DB console's](ui-overview.html) [**Jobs**](ui-jobs-page.html) page and [`SHOW JOBS`](show-jobs.html) SQL statements using the job ID.
+
+## Monitor a changefeed
+
+{{site.data.alerts.callout_info}}
+Monitoring is only available for Enterprise changefeeds.
```
Comment (ianjevans) [inline v21.2/monitor-and-debug-changefeeds.md:12]:
```suggestion
Monitoring is only available for {{ site.data.products.enterprise }} changefeeds.
```

File: v21.2/monitor-and-debug-changefeeds.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+{{site.data.alerts.callout_info}}
+You can use the high-water timestamp to [start a new changefeed where another ended](create-changefeed.html#start-a-new-changefeed-where-another-ended).
+{{site.data.alerts.end}}
+
+## Debug a changefeed
+
+### Using logs
+
+For Enterprise changefeeds, [use log information](logging-overview.html) to debug connection issues (i.e., `kafka: client has run out of available brokers to talk to (Is your cluster reachable?)`). Debug by looking for lines in the logs with `[kafka-producer]` in them:
```
Comment (ianjevans) [inline v21.2/monitor-and-debug-changefeeds.md:42]:
```suggestion
For {{ site.data.products.enterprise }} changefeeds, [use log information](logging-overview.html) to debug connection issues (i.e., `kafka: client has run out of available brokers to talk to (Is your cluster reachable?)`). Debug by looking for lines in the logs with `[kafka-producer]` in them:
```

File: v21.2/monitor-and-debug-changefeeds.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+I190312 18:56:53.535646 585 vendor/github.com/Shopify/sarama/client.go:123  [kafka-producer] Initializing new client
+I190312 18:56:53.535714 585 vendor/github.com/Shopify/sarama/client.go:724  [kafka-producer] client/metadata fetching metadata for all topics from broker localhost:9092
+I190312 18:56:53.536730 569 vendor/github.com/Shopify/sarama/broker.go:148  [kafka-producer] Connected to broker at localhost:9092 (unregistered)
+I190312 18:56:53.537661 585 vendor/github.com/Shopify/sarama/client.go:500  [kafka-producer] client/brokers registered new broker #0 at 172.16.94.87:9092
+I190312 18:56:53.537686 585 vendor/github.com/Shopify/sarama/client.go:170  [kafka-producer] Successfully initialized new client
+~~~
+
+### Using `SHOW CHANGEFEED JOBS`
+
+<span class="version-tag">New in v21.2:</span> For Enterprise changefeeds, use `SHOW CHANGEFEED JOBS` to check the status of your changefeed jobs:
```
Comment (ianjevans) [inline v21.2/monitor-and-debug-changefeeds.md:54]:
```suggestion
<span class="version-tag">New in v21.2:</span> For {{ site.data.products.enterprise }} changefeeds, use `SHOW CHANGEFEED JOBS` to check the status of your changefeed jobs:
```

File: v21.2/sql-statements.md [hunk 4]
```diff
@@ -196,7 +196,7 @@ Statement | Usage
 
 ## Changefeed statements (Enterprise)
 
-[Change data capture](stream-data-out-of-cockroachdb-using-changefeeds.html) (CDC) provides an Enterprise and core version of row-level change subscriptions for downstream processing.
+[Change data capture](change-data-capture-overview.html) (CDC) provides an Enterprise and core version of row-level change subscriptions for downstream processing.
```
Comment (ianjevans) [inline v21.2/sql-statements.md:199]:
```suggestion
[Change data capture](change-data-capture-overview.html) (CDC) provides an {{ site.data.products.enterprise }} and core version of row-level change subscriptions for downstream processing.
```

File: v21.2/sql-statements.md [hunk 5]
```diff
@@ -196,7 +196,7 @@ Statement | Usage
 
 ## Changefeed statements (Enterprise)
```
Comment (ianjevans) [inline v21.2/sql-statements.md:197]:
```suggestion
## Changefeed statements ({{ site.data.products.enterprise }})
```

File: v21.2/ui-jobs-page.md [hunk 6]
```diff
@@ -11,7 +11,7 @@ The **Jobs** page of the DB Console provides details about long-running tasks pe
 - Enterprise [`BACKUP`](backup.html) and [`RESTORE`](restore.html).
```
Comment (ianjevans) [inline v21.2/ui-jobs-page.md:11]:
```suggestion
- {{ site.data.products.enterprise }} [`BACKUP`](backup.html) and [`RESTORE`](restore.html).
```

File: releases/v21.2.0-beta.4.md [hunk 7]
```diff
@@ -33,7 +33,7 @@ $ docker pull cockroachdb/cockroach-unstable:v21.2.0-beta.4
 
 ### Enterprise edition changes
```
Comment (ianjevans) [inline releases/v21.2.0-beta.4.md:34]:
```suggestion
### {{ site.data.products.enterprise }} edition changes
```

File: releases/v21.2.0.md [hunk 8]
```diff
@@ -95,7 +95,7 @@ Core       | **Contention views**           | You can now use pre-built contenti
 Version    | Feature                                       | Description
 -----------+-----------------------------------------------+------------
 Enterprise | **`BACKUP` / `RESTORE` scalability**          | [`BACKUP`](../v21.2/backup.html) and [`RESTORE`](../v21.2/restore.html) performance has been improved for larger data volumes, more frequent backups, and clusters with more or larger nodes.
```
Comment (ianjevans) [inline releases/v21.2.0.md:97]:
```suggestion
{{ site.data.products.enterprise }} | **`BACKUP` / `RESTORE` scalability**          | [`BACKUP`](../v21.2/backup.html) and [`RESTORE`](../v21.2/restore.html) performance has been improved for larger data volumes, more frequent backups, and clusters with more or larger nodes.
```

File: releases/v21.2.0.md [hunk 9]
```diff
@@ -95,7 +95,7 @@ Core       | **Contention views**           | You can now use pre-built contenti
...
-Enterprise | **Webhook changefeed sink** | You can now stream individual [changefeed](../v21.2/create-changefeed.html) messages as webhook messages to a newly supported [`webhook-https` sink](../v21.2/create-changefeed.html#webhook-sink). The webhook sink is a flexible, general-purpose sink solution that does not require managing a Kafka cluster or cloud storage sink.
+Enterprise | **Webhook changefeed sink** | You can now stream individual [changefeed](../v21.2/create-changefeed.html) messages as webhook messages to a newly supported [`webhook-https` sink](../v21.2/changefeed-sinks.html#webhook-sink). The webhook sink is a flexible, general-purpose sink solution that does not require managing a Kafka cluster or cloud storage sink.
```
Comment (ianjevans) [inline releases/v21.2.0.md:98]:
```suggestion
{{ site.data.products.enterprise }} | **Webhook changefeed sink** | You can now stream individual [changefeed](../v21.2/create-changefeed.html) messages as webhook messages to a newly supported [`webhook-https` sink](../v21.2/changefeed-sinks.html#webhook-sink). The webhook sink is a flexible, general-purpose sink solution that does not require managing a Kafka cluster or cloud storage sink.
```

File: releases/v21.2.0.md [hunk 10]
```diff
@@ -95,7 +95,7 @@ Core       | **Contention views**           | You can now use pre-built contenti
...
 Enterprise | **Multi-region bulk operations improvements** | The following bulk operations are now supported: <ul><li>[`RESTORE`](../v21.2/restore.html) on [`REGIONAL BY TABLE`](../v21.2/multiregion-overview.html#regional-tables) and [`GLOBAL`](../v21.2/set-locality.html#global) tables is supported with some limitations. For more details, see [Restoring to multi-region databases](../v21.2/restore.html#restoring-to-multi-region-databases).</li><li>[`IMPORT INTO`](../v21.2/import-into.html) now supports importing into [`REGIONAL BY ROW`](../v21.2/set-locality.html#regional-by-row) tables.</li></ul>
```
Comment (ianjevans) [inline releases/v21.2.0.md:99]:
```suggestion
{{ site.data.products.enterprise }} | **Multi-region bulk operations improvements** | The following bulk operations are now supported: <ul><li>[`RESTORE`](../v21.2/restore.html) on [`REGIONAL BY TABLE`](../v21.2/multiregion-overview.html#regional-tables) and [`GLOBAL`](../v21.2/set-locality.html#global) tables is supported with some limitations. For more details, see [Restoring to multi-region databases](../v21.2/restore.html#restoring-to-multi-region-databases).</li><li>[`IMPORT INTO`](../v21.2/import-into.html) now supports importing into [`REGIONAL BY ROW`](../v21.2/set-locality.html#regional-by-row) tables.</li></ul>
```

File: releases/v21.2.0.md [hunk 11]
```diff
@@ -95,7 +95,7 @@ Core       | **Contention views**           | You can now use pre-built contenti
...
 Enterprise | **Changefeeds for regional by row tables**    | [Changefeeds](../v21.2/create-changefeed.html) are now supported on [regional by row tables](../v21.2/multiregion-overview.html#regional-by-row-tables).
```
Comment (ianjevans) [inline releases/v21.2.0.md:100]:
```suggestion
{{ site.data.products.enterprise }} | **Changefeeds for regional by row tables**    | [Changefeeds](../v21.2/create-changefeed.html) are now supported on [regional by row tables](../v21.2/multiregion-overview.html#regional-by-row-tables).
```

File: releases/v21.2.0.md [hunk 12]
```diff
@@ -95,7 +95,7 @@ Core       | **Contention views**           | You can now use pre-built contenti
...
 Enterprise | **Changefeed observability**                  | You can now display specific fields related to changefeed jobs by running [`SHOW CHANGEFEED JOBS`](../v21.2/show-jobs.html#show-changefeed-jobs).
```
Comment (ianjevans) [inline releases/v21.2.0.md:101]:
```suggestion
{{ site.data.products.enterprise }} | **Changefeed observability**                  | You can now display specific fields related to changefeed jobs by running [`SHOW CHANGEFEED JOBS`](../v21.2/show-jobs.html#show-changefeed-jobs).
```

File: releases/v21.2.4.md [hunk 13]
```diff
@@ -40,7 +40,7 @@ $ docker pull cockroachdb/cockroach:v21.2.4
 
 ### Enterprise edition changes
```
Comment (ianjevans) [inline releases/v21.2.4.md:41]:
```suggestion
### {{ site.data.products.enterprise }} edition changes
```

File: v21.2/change-data-capture-overview.md [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+
+Change data capture (CDC) provides efficient, distributed, row-level changefeeds into a configurable sink for downstream processing such as reporting, caching, or full-text indexing.
+
+## What is change data capture?
+
+While CockroachDB is an excellent system of record, it also needs to coexist with other systems. For example, you might want to keep your data mirrored in full-text indexes, analytics engines, or big data pipelines.
+
+The main feature of CDC is the changefeed, which targets an allowlist of tables, called the "watched rows". There are two implementations of changefeeds:
+
+| [Core changefeeds](changefeed-examples.html#create-a-core-changefeed)   | [Enterprise changefeeds](changefeed-examples.html) |
```
Comment (ianjevans) [inline v21.2/change-data-capture-overview.md:15]:
```suggestion
| [Core changefeeds](changefeed-examples.html#create-a-core-changefeed)   | [{{ site.data.products.enterprise }} changefeeds](changefeed-examples.html) |
```

File: v21.2/change-data-capture-overview.md [hunk 15]
```diff
@@ -0,0 +1,35 @@
...
+--------------------------------------------------|-----------------------------------------------------------------|
+| Useful for prototyping or quick testing. | Recommended for production use. |
+| Available in all products. | Available in {{ site.data.products.dedicated }} or with an [Enterprise license](enterprise-licensing.html) in CockroachDB. |
```
Comment (ianjevans) [inline v21.2/change-data-capture-overview.md:18]:
```suggestion
| Available in all products. | Available in {{ site.data.products.dedicated }} or with an [{{ site.data.products.enterprise }} license](enterprise-licensing.html) in CockroachDB. |
```

File: v21.2/changefeed-examples.md [hunk 16]
```diff
@@ -0,0 +1,671 @@
...
+
+### Create a changefeed connected to Kafka using Avro
+
+{{site.data.alerts.callout_info}}
+[`CREATE CHANGEFEED`](create-changefeed.html) is an [Enterprise-only](enterprise-licensing.html) feature. For the Core version, see [the `CHANGEFEED FOR` example](#create-a-core-changefeed-using-avro).
+{{site.data.alerts.end}}
+
+In this example, you'll set up a changefeed for a single-node cluster that is connected to a Kafka sink and emits [Avro](https://avro.apache.org/docs/1.8.2/spec.html) records. The changefeed will watch two tables.
+
+1. If you do not already have one, [request a trial Enterprise license](enterprise-licensing.html).
```
Comment (ianjevans) [inline v21.2/changefeed-examples.md:243]:
```suggestion
1. If you do not already have one, [request a trial {{ site.data.products.enterprise }} license](enterprise-licensing.html).
```

File: v21.2/changefeed-examples.md [hunk 17]
```diff
@@ -0,0 +1,671 @@
...
+
+### Create a changefeed connected to a cloud storage sink
+
+{{site.data.alerts.callout_info}}
+[`CREATE CHANGEFEED`](create-changefeed.html) is an [Enterprise-only](enterprise-licensing.html) feature. For the Core version, see [the `CHANGEFEED FOR` example above](#create-a-core-changefeed).
+{{site.data.alerts.end}}
+
+In this example, you'll set up a changefeed for a single-node cluster that is connected to an AWS S3 sink. The changefeed watches two tables. Note that you can set up changefeeds for any of [these cloud storage providers](changefeed-sinks.html#cloud-storage-sink).
+
+1. If you do not already have one, [request a trial Enterprise license](enterprise-licensing.html).
```
Comment (ianjevans) [inline v21.2/changefeed-examples.md:440]:
```suggestion
1. If you do not already have one, [request a trial {{ site.data.products.enterprise }} license](enterprise-licensing.html).
```

File: v21.2/changefeed-examples.md [hunk 18]
```diff
@@ -0,0 +1,671 @@
...
+
+{{site.data.alerts.callout_info}}
+[`CREATE CHANGEFEED`](create-changefeed.html) is an [enterprise-only](enterprise-licensing.html) feature. For the Core version, see [the `CHANGEFEED FOR` example above](#create-a-core-changefeed).
+{{site.data.alerts.end}}
+
+{% include {{ page.version.version }}/cdc/webhook-beta.md %}
+
+<span class="version-tag">New in v21.2:</span> In this example, you'll set up a changefeed for a single-node cluster that is connected to a local HTTP server via a webhook. For this example, you'll use an [example HTTP server](https://github.com/cockroachlabs/cdc-webhook-sink-test-server/tree/master/go-https-server) to test out the webhook sink.
+
+1. If you do not already have one, [request a trial enterprise license](enterprise-licensing.html).
```
Comment (ianjevans) [inline v21.2/changefeed-examples.md:563]:
```suggestion
1. If you do not already have one, [request a trial {{ site.data.products.enterprise }} license](enterprise-licensing.html).
```

File: v21.2/create-and-configure-changefeeds.md [hunk 19]
```diff
@@ -0,0 +1,80 @@
+---
+title: Create and Configure Changefeeds
+summary: Create and configure a changefeed job for core and enterprise.
```
Comment (ianjevans) [inline v21.2/create-and-configure-changefeeds.md:3]:
```suggestion
summary: Create and configure a changefeed job for Core and Enterprise.
```

File: v21.2/licensing-faqs.md [hunk 20]
```diff
@@ -56,8 +56,8 @@ Feature          | BSL | CCL (free)      | CCL (paid)
 **[Full backups](take-full-and-incremental-backups.html#full-backups)** | | ✓ |
 **[Incremental backups](take-full-and-incremental-backups.html#incremental-backups)** | | | ✓
 **[Other advanced backup features](backup.html)** | | | ✓
-**[Core changefeed](stream-data-out-of-cockroachdb-using-changefeeds.html#create-a-core-changefeed)** | | ✓ |
-**[Enterprise changefeed](stream-data-out-of-cockroachdb-using-changefeeds.html#configure-a-changefeed-enterprise)** | | | ✓
+**[Core changefeed](create-and-configure-changefeeds.html#create-a-changefeed-core)** | | ✓ |
+**[Enterprise changefeed](create-and-configure-changefeeds.html#configure-a-changefeed-enterprise)** | | | ✓
```
Comment (ianjevans) [inline v21.2/licensing-faqs.md:60]:
```suggestion
**[{{ site.data.products.enterprise }} changefeed](create-and-configure-changefeeds.html#configure-a-changefeed-enterprise)** | | | ✓
```

File: releases/v21.2.0-beta.1.md [hunk 21]
```diff
@@ -75,32 +75,32 @@ $ docker pull cockroachdb/cockroach-unstable:v21.2.0-beta.1
 
 - Added new `DEBUG_PAUSE_ON` option to [`RESTORE`](../v21.2/functions-and-operators.html) jobs to allow for self pause on errors. [#69422][#69422]
 - [Changefeed option](../v21.2/create-changefeed.html#options) values are now case insensitive. [#69217][#69217]
-- Performance for [changefeeds](../v21.2/stream-data-out-of-cockroachdb-using-changefeeds.html) during some range-split operations is now improved. [#66312][#66312]
-- [Cloud storage sinks for enterprise changefeeds](../v21.2/create-changefeed.html#cloud-storage-sink) are no longer experimental. [#69787][#69787]
+- Performance for [changefeeds](../v21.2/change-data-capture-overview.html) during some range-split operations is now improved. [#66312][#66312]
+- [Cloud storage sinks for enterprise changefeeds](../v21.2/create-changefeed.html#cloud-storage) are no longer experimental. [#69787][#69787]
```
Comment (ianjevans) [inline releases/v21.2.0-beta.1.md:79]:
```suggestion
- [Cloud storage sinks for {{ site.data.products.enterprise }} changefeeds](../v21.2/create-changefeed.html#cloud-storage) are no longer experimental. [#69787][#69787]
```

File: v21.2/change-data-capture-overview.md [hunk 22]
```diff
@@ -0,0 +1,37 @@
...
+| [`CREATE`](create-and-configure-changefeeds.html#create-a-changefeed-core) changefeed and cancel by closing the connection. | Manage changefeed with [`CREATE`](create-and-configure-changefeeds.html#create), [`PAUSE`](create-and-configure-changefeeds.html#pause), [`RESUME`](create-and-configure-changefeeds.html#resume), and [`CANCEL`](create-and-configure-changefeeds.html#cancel), as well as [monitor](monitor-and-debug-changefeeds.html#monitor-a-changefeed) and [debug](monitor-and-debug-changefeeds.html#debug-a-changefeed). |
+
+See [Ordering Guarantees](use-changefeeds.html#ordering-guarantees) for detail on CockroachDB's at-least-once-delivery-guarantee as well as explanation on how rows are emitted.
+
+See [Ordering Guarantees](use-changefeeds.html#ordering-guarantees) for detail on CockroachDB's at-least-once-delivery-guarantee as well as explanation on how rows are emitted.
```
Comment (amruss) [inline v21.2/change-data-capture-overview.md:26]: This paragraph is repeated twice

**Review by amruss — APPROVED** [review]
I had a minor nit comment but other than that LGTM

**Review by ianjevans — APPROVED** [review]

**Comment by kathancox** [comment]
TFTRs @ianjevans & @amruss!

**Pull request merged**